lisp/ess-custom.el

```diff
@@ -646,211 +646,345 @@ regardless of where in the line point is when the TAB command is used."
 nil means to use R/S indentation.")
 (make-variable-buffer-local 'ess-indent-line-function)
 
-(defvar ess-indent-level 2
+(defvar ess-indent-level 4
   "Indentation of S statements with respect to containing block.")
 
-(defvar ess-brace-imaginary-offset 0
-  "Imagined indentation of an open brace following a statement.")
+(defvar ess-offset-statement '(t)
```

Comment:
you're right this would be much better. I used `statement` because this setting also applies to expressions wrapped in parentheses and I didn't think of those as blocks since they can only contain one expression. But in the end this doesn't matter much. I'll rename to `block`